collection-spec/json-schema/collection.json

```diff
@@ -97,7 +90,18 @@
             }
           }
         }
-      }
+      },
+      "required": [
+        "esmcat_version",
+        "id",
+        "description",
+        "attributes",
+        "assets"
+      ],
+      "oneOf": [
+        { "required": ["catalog_file"] },
+        { "required": ["catalog_dict"] }
+      ]
```

Comment:
> Are both catalog_file and catalog_dict required?

They are mutually exclusive. Should I remove the `REQUIRED` statement from the table to avoid confusion?

Reply:
That was what tripped me up. Up to you on how to treat this part of the schema.